src/AISmart.Domain/Twitter/BindTwitterAccountDto.cs

```diff
@@ -0,0 +1,8 @@
+namespace AISmart.Twitter;
+
+public class BindTwitterAccountDto
+{
+    public string UserId { get; set; }
+    public string Token { get; set; }
+    public string TokenSecret { get; set; }
```

Comment:
Seems like you have encrypted this already. I think it would be better to name it as encryptedTokenSecret to avoid confusion.